pyproject.toml

```diff
@@ -55,11 +55,11 @@ PyYAML = "^6.0"
 defusedxml = "^0.7.1"
 colorama = "^0.4.6"
 packaging = ">=23.0"
-splunk-appinspect = { version = "^3.9.1", extras = ["validate"], optional = true }
+splunk-appinspect = { version = "^3.10.0", optional = true, python = ">=3.9"}
+pillow = { version = "10.4.0", optional = true, python = ">=3.9"}
```

Comment:
I do not think we need to have this declared explicitly.

Reply:
I think we need it because `splunk-appinspect` allows a range of versions for `pillow` library, hence we would have to explicitly tell it that we need a non-vulnerable version, which is 10.4.0. Moreover, there is another failure for `setuptools` as well in Fossa, hence I think have the same for it also.

Reply:
I reached out to AppInspect team in Slack.